src/panels/config/entities/dialog-entity-editor.ts

```diff
@@ -87,88 +83,89 @@ export class DialogEntityEditor extends LitElement {
-          <div class="main-title" main-title>
-            ${stateObj ? computeStateName(stateObj) : entry?.name || entityId}
-          </div>
-          ${stateObj
-            ? html`
-                <ha-icon-button
-                  aria-label=${this.hass.localize(
-                    "ui.dialogs.entity_registry.control"
-                  )}
-                  icon="hass:tune"
-                  @click=${this._openMoreInfo}
-                ></ha-icon-button>
-              `
-            : ""}
-        </app-toolbar>
-        <paper-tabs
-          scrollable
-          hide-scroll-buttons
-          .selected=${this._curTabIndex}
-          @selected-item-changed=${this._handleTabSelected}
-        >
-          <paper-tab id="tab-settings">
-            ${this.hass.localize("ui.dialogs.entity_registry.settings")}
-          </paper-tab>
-          ${Object.entries(this._extraTabs).map(
-            ([key, tab]) => html`
-              <paper-tab id=${key}>
-                ${this.hass.localize(tab.translationKey) || key}
-              </paper-tab>
-            `
-          )}
-          <paper-tab id="tab-related">
-            ${this.hass.localize("ui.dialogs.entity_registry.related")}
-          </paper-tab>
-        </paper-tabs>
-        ${cache(
-          this._curTab === "tab-settings"
-            ? entry
-              ? this._settingsElementTag
-                ? html`
-                    ${dynamicElement(this._settingsElementTag, {
-                      hass: this.hass,
-                      entry,
-                      entityId,
-                      dialogElement: this._dialog,
-                    })}
-                  `
-                : ""
-              : html`
-                  <paper-dialog-scrollable>
-                    ${this.hass.localize(
-                      "ui.dialogs.entity_registry.no_unique_id"
+        <div slot="heading">
+          <app-toolbar>
+            <mwc-icon-button
+              .label=${this.hass.localize("ui.dialogs.entity_registry.dismiss")}
+              dialogAction="cancel"
+            >
+              <ha-svg-icon .path=${mdiClose}></ha-svg-icon>
+            </mwc-icon-button>
+            <div class="main-title" main-title>
+              ${stateObj ? computeStateName(stateObj) : entry?.name || entityId}
+            </div>
+            ${stateObj
+              ? html`
+                  <mwc-icon-button
+                    .label=${this.hass.localize(
+                      "ui.dialogs.entity_registry.control"
                     )}
-                  </paper-dialog-scrollable>
+                    @click=${this._openMoreInfo}
+                  >
+                    <ha-svg-icon .path=${mdiTune}></ha-svg-icon>
+                  </mwc-icon-button>
                 `
-            : this._curTab === "tab-related"
-            ? html`
-                <paper-dialog-scrollable>
+              : ""}
+          </app-toolbar>
+          <paper-tabs
```

Comment:
One day we should kill this too :)

Reply:
yeah, not sure if the side scrolling in `mwc` is fixed already